### ISSUE TITLE
Depthwise separable 2D convolution

### ISSUE THREAD

**Issue by avitalsh** [tf-encrypted/tf-encrypted#460]
Hi,

Is there a plan to add support for depthwise separable 2D convolution (DepthwiseConv2D) ? 

Thanks

**Comment by jvmancuso** [comment]
Yes, it's on our roadmap, although we've prioritized a few other items (in particular, more robust support for Keras layers and models).  @yanndupis or I have some other things on our roadmap ahead of that, but we hoped to get to it in the next couple weeks.

If you'd like to submit a PR adding it to the Pond protocol, we'd be more than happy to work with you to merge it!

**Comment by mortendahl** [comment]
@jvmancuso is this something we should bump up and look at within a week or two?

**Comment by jvmancuso** [comment]
This would be great to work on after the keras work is done. The forward pass is easy, but the backward pass can be quite tricky.

**Comment by avitalsh** [comment]
@jvmancuso Hey, is this planned to be added sometime soon?

**Comment by jvmancuso** [comment]
We've had plans to do so, but it's been a bit neglected -- would you find it useful to have it?

If you'd like to add it, we'd be more than happy to help you with the implementation and get it merged.
